packages/devtools/src/launcher.js

```diff
@@ -58,6 +57,7 @@ async function launchChrome (capabilities) {
 }
 
 function launchFirefox (capabilities) {
+    puppeteerFirefox = require('puppeteer-firefox');
```

Comment:
There are still some linter issues:

```
/home/travis/build/webdriverio/webdriverio/packages/devtools/src/launcher.js
  60:5   error  'puppeteerFirefox' is not defined  no-undef
  60:52  error  Extra semicolon                    semi
  62:12  error  'puppeteerFirefox' is not defined  no-undef
```

Please fix by
```suggestion
    const puppeteerFirefox = require('puppeteer-firefox')
```